Doc/library/gettext.rst

```diff
@@ -669,7 +669,7 @@ implementations, and valuable experience to the creation of this module:
 .. [#] The default locale directory is system dependent; for example, on RedHat Linux
    it is :file:`/usr/share/locale`, but on Solaris it is :file:`/usr/lib/locale`.
    The :mod:`gettext` module does not try to support these system dependent
-   defaults; instead its default is :file:`sys.prefix/share/locale`. For this
+   defaults; instead its default is :file:`{sys.prefix}/share/locale`. For this
```

Comment:
I wonder if it would be useful to rework that line to add a linked reference to sys.prefix.
Maybe adding ```(see :data:`sys.prefix`)``` at the end of the sentence?

Reply:
why not, can be useful.

Reply:
done